Makefile

```diff
@@ -191,7 +191,7 @@ install: profile $(shell find proteus -type f) $(wildcard *.py) proteus
 	@echo "done installing standard extension modules"
 	@echo "************************"
 	@echo "installing scripts"
-	cd scripts && ${PROTEUS_ENV} PROTEUS_PREFIX=${PROTEUS_PREFIX} make
+	#cd scripts && ${PROTEUS_ENV} PROTEUS_PREFIX=${PROTEUS_PREFIX} make
```

Comment:
Uncommented in commit 6696689b85b7f7333e491b25931f5b43cee1bc53. There was an issue in building the scripts on SCOREC machines because the f77 Fortran compiler was not available, so it was temporarily turned off to avoid the issue.